lightning/src/util/transaction_utils.rs

```diff
@@ -226,30 +226,29 @@ mod tests {
 	#[test]
 	fn test_tx_change_edge() {
 		// Check that we never add dust outputs
-		let mut tx = Transaction { version: 2, lock_time: LockTime::ZERO, input: Vec::new(), output: Vec::new() };
+		let mut tx = Transaction { version: Version::TWO, lock_time: LockTime::ZERO, input: Vec::new(), output: Vec::new() };
 		let orig_wtxid = tx.wtxid();
 		let output_spk = ScriptBuf::new_p2pkh(&PubkeyHash::hash(&[0; 0]));
 		assert_eq!(output_spk.dust_value().to_sat(), 546);
-		// 9 sats isn't enough to pay fee on a dummy transaction...
-		assert_eq!(tx.weight().to_wu(), 40); // ie 10 vbytes
-		assert!(maybe_add_change_output(&mut tx, 9, 0, 250, output_spk.clone()).is_err());
+		assert_eq!(tx.weight().to_wu(), 42); // 3 * (base size = 4 + 1 + 1 + 4 = 10) + (total size = 4 + 2 + 1 + 1 + 4 = 12)
```

Comment:
We should label the individual terms in the base size and total size calculation comment here for clarity.